src/main/java/net/sf/jabref/logic/autocompleter/NameFieldAutoCompleter.java

```diff
@@ -93,7 +94,7 @@ public void addBibtexEntry(BibEntry entry) {
             if (entry.hasField(fieldName)) {
                 String fieldValue = entry.getField(fieldName);
                 AuthorList authorList = AuthorList.getAuthorList(fieldValue);
-                for (AuthorList.Author author : authorList.getAuthorList()) {
+                for (Author author : authorList.getAuthorList()) {
```

Comment:
OK! Sounds good! I use some tool on the phone to check it so maybe that has
problems. I can for example not see your comment in the tool...